server/src/pages/shotindex/server.js

```diff
@@ -18,7 +18,7 @@ app.get("/", function(req, res) {
   }
   getShots.then(_render)
     .catch((err) => {
-      res.type("txt").status(500).send("Error rendering page: " + err);
+      res.type("txt").status(500).send("Error rendering page: " + err); // todo l10n: shotIndexPageErrorRendering
```

Comment:
Here is another edge case I'm not sure about: in some cases, we render a server error string directly into the page.

I've been thinking I could work around this by passing an l10n ID into the page, then running fluent-react on that page, to swap in the appropriate error message. Is that the best approach? Is there a fluent library that could be used instead, for spots like this?